src/components/algolia/algolia-course-related.tsx

```diff
@@ -0,0 +1,84 @@
+"use client"
+
+import {useMemo} from "react"
+import algoliasearch from "algoliasearch/lite"
+import {RelatedProducts} from "react-instantsearch"
+import {InstantSearchNext} from "react-instantsearch-nextjs"
+import ImageCard from "@components/patterns/image-card"
+import {H3} from "@components/elements/headers"
+import ActionLink from "@components/elements/action-link"
+
+type Props = {
+  objectID: string
+  appId: string
+  searchIndex: string
+  searchApiKey: string
+}
+
+type RelatedCourseItemProps = {
+  sum_course_catalog_number?: string
+  sum_course_class_number?: string
+  title?: string
+  photo?: string
+  url?: string
+}
+
+const RelatedCourseItem = ({item}: {item: RelatedCourseItemProps}) => {
+  return (
+    <ImageCard
+      imageUrl={item.photo}
+      imageAlt=""
+    >
+      <div className="flex flex-col">
+        <H3 className="rs-mb-0 order-2 max-w-[900px] font-roboto font-normal">{item.title}</H3>
+        <div className="rs-mb-0 order-1 font-normal">{item.sum_course_catalog_number}</div>
+        {item.url && (
+          <ActionLink
+            className="order-3 font-roboto text-18 font-medium no-underline hocus:underline"
+            href={item.url}
+          >
+            Learn more
+          </ActionLink>
+        )}
+      </div>
+    </ImageCard>
+  )
+}
+
+const RelatedCourses = ({objectID, appId, searchIndex, searchApiKey}: Props) => {
+  const searchClient = useMemo(() => algoliasearch(appId, searchApiKey), [appId, searchApiKey])
+
+  return (
+    <InstantSearchNext
+      indexName={searchIndex}
+      searchClient={searchClient}
+      future={{preserveSharedStateOnUnmount: true}}
+    >
+      <RelatedProducts
```

Comment:
Do you want to customize the [empty message](https://www.algolia.com/doc/api-reference/widgets/related-products/react/#widget-param-emptycomponent)